### DISCUSSION TITLE
Image not loaded when I open the edit page of the nova resource, getting cors error

### DISCUSSION THREAD

**Discussion by AfsarKetto** [laravel/nova-issues#6008] [Q&A]
- Laravel Version: 9.0
- Nova Version: ~4.0
- PHP Version: 8.4
- Browser type and version: Chrome and Firefox

When open the edit page image not loaded getting cors error.But Image is loaded in details page issue is edit page. What a do for this issue.

![Screenshot from 2023-10-31 12-25-08](https://github.com/laravel/nova-issues/assets/149465639/4fad06a0-a89b-4be0-a53f-c82f13b0cf52)

**Comment by crynobone** [comment]
Yes, edit page load the image using JavaScript and incorrect CORS will cause an issue here. The correct solution is to use the correct CORS configuration.

**Reply by AfsarKetto** [reply]
I am Using CDN Not Server.

**Reply by crynobone** [reply]
Most CDN would still allow you to configure CORS.

**Comment by BarneyMayerson** [comment]
@AfsarKetto You can add 
`Header always set Access-Control-Allow-Origin "*"` 
into `.htaccess` file. 
Restart your browser.
After you may comment this string. It still be worked.

**Comment by Aakashjoshi98** [comment]
@crynobone 
i have already done all of the things, im using laravel-cors package, but i'm still stuck in it
